packages/circle-demo-webapp/package.json

```diff
@@ -14,6 +14,7 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@chainsafe/eslint-config": "^2.2.4",
```

Comment:
why adding this if it is not used anywhere? I don't see this added to eslint configuration file